packages/core/src/lib/toast/context.ts

```diff
@@ -0,0 +1,141 @@
+import { setContext, getContext } from 'svelte';
+import {
+  derived,
+  get,
+  writable,
+  type Readable,
+  type Writable,
+} from 'svelte/store';
+
+import { uniqueId } from '$lib/unique-id';
+import { ToastVariant, type ToastVariantType } from '$lib/toast';
+
+import { pausableProgress } from '../notification/pausable-progress';
+
+/** Internal toast context. */
+export interface ToastContext {
+  state: ToastState;
+  toast: Toast;
+}
+
+/** Notification sender. */
+export interface Toast {
+  /** Push a neutral toast with a message. */
+  neutral: (message: string, closeable?: boolean) => void;
```

Comment:
Same question as below with `upload/process`. Our immediate needs with toasts are all success toasts. Let's not add anything in this PR that we aren't going to use immediately